Object.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Object.swift
+//  Organization
+//
+//  Created by Caio on 22.10.20.
+//  Copyright © 2020. All rights reserved.
+//
+
+import Foundation
+
+struct Object {
+    let id: Int
+    let name: String
+    let description: String
+
+    func doSomething() {
+        let sum = "1 + 1 = 3"
+        print(sum)
+    }
```

Comment:
The `doSomethingElse` function is missing:

```suggestion
    }
    
    func doSomethingElse() {
        print("Something else is done")
    }
```